models/mart/localbike_dataset/mrt_monthly_order_report.sql

```diff
@@ -9,7 +9,6 @@ SELECT
   t.first_name,
   t.last_name,
   t.manager_id,
-  -- comments

```

Comment:
first_name & last_name could be maintained and concatenate on the same field which will allow to make analysis on the staff level and so on.